### PR TITLE
[Test Controller] New test controller applied to Compute

### PR DESCRIPTION
**Pull request by vladimir-shcherbakov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Compute/Compute.Test/ScenarioTests/ComputeTestRunner.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    @"ComputeTestCommon.ps1",
+                    @"../AzureRM.Resources.ps1",
+                    @"../AzureRM.Storage.ps1"
+                })
+                .WithNewRmModules (helper => new[]
+                {
+                    helper.RMProfileModule,
+#if !NETSTANDARD
+                    helper.RMStorageDataPlaneModule,
+#endif
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/ComputeTestRunner.cs]: Please delete this #if statement as it is not used.

File: src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.cs [hunk 2]
```diff
@@ -12,42 +12,39 @@
...
-            XunitTracingInterceptor.AddToContext(_logger);
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestExportLogAnalyticThrottledRequestsNegative()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-ExportLogAnalyticThrottledRequestsNegative");
+            TestRunner.RunTestScript("Test-ExportLogAnalyticThrottledRequestsNegative");
+
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.cs]: Excess carriage return.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.cs [hunk 3]
```diff
@@ -18,61 +18,58 @@
...
-            XunitTracingInterceptor.AddToContext(_logger);
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVm()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVm");
+            TestRunner.RunTestScript("Test-SimpleNewVm");
+
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.cs]: Excess carriage return.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 4]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmss()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmss");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmss");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 5]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
...
+            TestRunner.RunTestScript("Test-SimpleNewVmss");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmssFromSIGImage()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssFromSIGImage");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssFromSIGImage");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 6]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
...
+            TestRunner.RunTestScript("Test-SimpleNewVmssFromSIGImage");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmssWithUltraSSD()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithUltraSSD");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithUltraSSD");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 7]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
...
+            TestRunner.RunTestScript("Test-SimpleNewVmssWithUltraSSD");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmssLbErrorScenario()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssLbErrorScenario");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssLbErrorScenario");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 8]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
...
+            TestRunner.RunTestScript("Test-SimpleNewVmssLbErrorScenario");
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmssWithSystemAssignedIdentity()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithSystemAssignedIdentity");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithSystemAssignedIdentity");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 9]
```diff
@@ -37,43 +38,49 @@ public StrategiesVmssTests(Xunit.Abstractions.ITestOutputHelper output)
...
+            TestRunner.RunTestScript("Test-SimpleNewVmssWithSystemAssignedIdentity");
         }
 
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSimpleNewVmssImageName()
         {
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssImageName");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssImageName");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

File: src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs [hunk 10]
```diff
@@ -95,38 +102,21 @@ public void TestSimpleNewVmssWithSystemAssignedUserAssignedIdentity()
              * Get-AzureRmUserAssignedIdentity -ResourceGroupName UAITG123456 -Name UAITG123456Identity
              * Nore down the Id and use it in the PS code
              * */
-            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithsystemAssignedUserAssignedIdentity");
+//            ComputeTestController.NewInstance.RunPsTest(_logger, "Test-SimpleNewVmssWithsystemAssignedUserAssignedIdentity");
```
Comment (MiYanni) [inline src/Compute/Compute.Test/ScenarioTests/StrategiesVmssTests.cs]: Remove commented code.

**Comment by vladimir-shcherbakov** [comment]
@hyonholee We are applying a new test controller - could you take a look?

**Comment by vladimir-shcherbakov** [comment]
@MiYanni  Are you ready to sign off?

**Comment by vladimir-shcherbakov** [comment]
signing: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign-esrp/119/

**Pull request merged**